design-system/gulpfile.js

```diff
@@ -25,7 +19,7 @@ const src = 'src/';
 const dest = 'build/';
 
 const src_paths = {
-  css: src + '_css/**/*.{pcss,css}',
+  css: src + '_css/*.{pcss,css}',
```

Comment:
We only need to compile the top-level CSS, the nested ones are loaded via `@import`